### PR TITLE
Fix custom domains docs structure

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

**Comment by pragmaticivan** [comment]
Can you review that doc please? @eduardolundgren @zenorocha

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

File: intro/custom-domains.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 
-###### This is an overview of how WeDeploy manage custom domains for your projects.
+###### This is an overview of how WeDeploy manages custom domains for your projects.
 
 <!-- <article id="introduction"> -->
 
 ## Introduction
 
-In order to make your app accessible right away, WeDeploy makes any project accessible via its project domain, wich looks like `<serviceID>.<projectID>.wedeploy.io`. Let's say you have a project named `project` with a service named `web` the WeDeploy project domain would be `web.project.wedeploy.io`. Every project in WeDeploy supports one or many custom domains.
+To make your app accessible right away, WeDeploy makes any project accessible via its project domain; wich looks like `<serviceID>.<projectID>.wedeploy.io`. Let's say you have a project named `project` with a service named `web` the WeDeploy project domain would be `web.project.wedeploy.io`. Every project in WeDeploy supports one or many custom domains.
```
Comment (eduardolundgren) [inline intro/custom-domains.md]: why using `;` instead of `,`?

File: intro/custom-domains.md [hunk 2]
```diff
@@ -31,15 +32,10 @@ To make a project accessible via one or more non-WeDeploy domain names, you must
 
 ## Configuring DNS for root domains
 
-<!-- ##### Fix that -->
-Root domains doesn't have subdomain on its structure. You can add root domains on the same project settings page used for domains with subdomains.
-Configuring your DNS provider for a root domain is similar to configuring a DNS provider for a CNAME. In this case, the type of record depends on the DNS provider.
+A root domain is the highest level of hierarchy for the website you control, ex: `wedeploy.com`. When you register a domain name, you are registering a root domain. This means you have the access to create subdomains and file structures all branching from that root domain.
```
Comment (eduardolundgren) [inline intro/custom-domains.md]: The example here should be `mydomain.com` like all the others.

File: intro/custom-domains.md (hunk 1 above)
Comment (zenorocha) [reply intro/custom-domains.md]: `wich` > `which`

Comment (pragmaticivan) [reply intro/custom-domains.md]: @eduardolundgren it was a suggestion from `https://grammarly.com/` I can change back if you want.

**Comment by pragmaticivan** [comment]
done @eduardolundgren .

File: intro/custom-domains.md (hunk 1 above)
Comment (eduardolundgren) [reply intro/custom-domains.md]: OK. We can use that. Thx.

**Pull request merged**